coalib/bearlib/__init__.py

```diff
@@ -97,3 +99,41 @@ def wrapping_function(*args, **kwargs):
         return wrapping_function
 
     return _deprecate_decorator
+
+
+def deprecate_bear(bear):
+    """
+    Use this to deprecate a bear. Say we have a bear:
+
+    >>> class SomeBear:
+    ...     def run(*args):
+    ...         print("I'm running!")
+
+    To change the name from ``SomeOldBear`` to ``SomeBear`` you can keep the
+    ``SomeOldBear.py`` around with those contents:
+
+    >>> @deprecate_bear
+    ... class SomeOldBear(SomeBear): pass
+
+    Now let's run the bear:
+
+    >>> import sys
+    >>> logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
+    >>> SomeOldBear().run()
+    WARNING:root:The bear SomeOldBear is deprecated. Use SomeBear instead!
+    I'm running!
+
+    :param bear: An old bear class that inherits from the new one (so it gets
+                 its methods and can just contain a pass.)
+    :return: A bear class that warns about deprecation on use.
```

Comment:
align the :return: docs with the one of the setting above 
anyway this can be a newcomer issue :sweat_smile: